staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -133,6 +133,8 @@ var (
 		kubectl apply --prune -f manifest.yaml --all --prune-whitelist=core/v1/ConfigMap`))
 
 	warningNoLastAppliedConfigAnnotation = "Warning: resource %[1]s is missing the %[2]s annotation which is required by %[3]s apply. %[3]s apply should only be used on resources created declaratively by either %[3]s create --save-config or %[3]s apply. The missing annotation will be patched automatically.\n"
+	// WarningChangesOnDeletingResource is for warning about the resource currently being deleted on which a kubectl operation is run.
```

Comment:
That comment is not useful. If it was, then it would probably mean that the message needs to be improved ;-)